integration/e2e/multi_tenant_test.go

```diff
@@ -228,12 +241,29 @@ func TestMultiTenantSearch(t *testing.T) {
 			require.NoError(t, err)
 			// actual error comes in resp, need to call Recv to get it.
 			_, grpcErr := grpcResp.Recv()
+			// fmt.Printf("==== grpcResp.Recv() grpcErr: %s\n", grpcErr)
+			// fmt.Printf("==== grpcResp.Recv() grpcSearchResp: %s\n", grpcSearchResp)
+
+			// drain the stream until everything is returned
+			// for {
+			// 	_, grpcErr = grpcResp.Recv()
+			// 	if errors.Is(err, io.EOF) {
+			// 		break
+			// 	}
+			// 	require.NoError(t, err)
+			// }
+
+			// find the trace with streaming. using the http server b/c that's what Grafana will do
+			// FIXME: info is for the last tenant so it's gonna fail, assert streaming in it's own test?? and only check for no error here
+			// rpc error: code = Unknown desc = no org id
+			// util.SearchStreamAndAssertTrace(t, grpcClient, info, now.Add(-20*time.Minute).Unix(), now.Unix())
 
 			if tc.tenantSize > 1 {
-				// we expect error in case of multi-tenant request for unsupported endpoints
+				// error for multi-tenant request for unsupported endpoints
 				require.Error(t, msErr)
-				require.Error(t, wsErr)
-				require.Error(t, grpcErr)
+				// no error for multi-tenant streaming search
```

Comment:
does this test assert the expected content? or just that there's no error?

Reply:
just no error, and not content.

I am adding more unit tests for multi-tenant streaming in `search_streaming_test.go`

Reply:
any particular reason not to? we use this method in other tests to do it:

https://github.com/grafana/tempo/blob/main/integration/util.go#L368

Reply:
updated the test to assert the content.